### PR TITLE
vue3: Added gallery view

### PR DESCRIPTION
**Pull request by mustafapc19**
Ref: #2631 

Issues:
Known Scrolling has an offset 
image

UI:
image

### CODE REVIEW

**Comment by o1lab** [comment]
Confirming if multiple images within a card mean - we can go through images at the card level?

File: packages/nc-gui-v2/components/smartsheet/Gallery.vue [hunk 1]
```diff
@@ -12,235 +13,86 @@ const isPublicView = false
...
-                        :sql-ui="sqlUi"
-                        :is-locked="isLocked"
-                        class="xc-input body-2"
-                        :meta="meta"
-                      /> -->
-                <!--                </v-col> -->
-                <!--              </v-row> -->
+<!-- TODO: Fix scrolling -->
+<template>
+  <div class="flex flex-col h-100 min-h-0 w-100">
```
Comment (bcakmakoglu) [inline packages/nc-gui-v2/components/smartsheet/Gallery.vue:57]: don't use `h-100` or `w-100` - those are vuetify styles.
Use `h-full` and `w-full` - TailwindCSS classes.

File: packages/nc-gui-v2/components/smartsheet/Gallery.vue [hunk 2]
```diff
@@ -12,235 +13,86 @@ const isPublicView = false
...
+    <div class="nc-gallery-container min-h-0 flex-1 grid grid-cols-4 gap-x-4 gap-y-4 my-4 px-3">
```
Comment (bcakmakoglu) [inline packages/nc-gui-v2/components/smartsheet/Gallery.vue:58]: `gap-x-4` and gap-y-4` are sort of redundant. Just use `gap-4`

**Review by bcakmakoglu — CHANGES_REQUESTED** [review]

**Comment by mustafapc19** [comment]
@o1lab No right now only the first image is shown as it is in v1. Should we add a carousal like flow to show multiple images?

**Comment by mustafapc19** [comment]
@o1lab I missed the image carousal in v1. Will add it here as well 👍

**Pull request merged**